File: airport_on_time_performance/v1/get/Node SDK/airport_on_time_performance.js
const WAConvo = require("waconvo");

const waconvo = new WAConvo({
  clientId: "YOUR_API_KEY",
  clientSecret: "YOUR_API_SECRET",
});

async function main() {
  try {
    // Will there be a delay in the JFK airport on the 1st of September?
    const response = await waconvo.airport.predictions.onTime.get({
      airportCode: "JFK",
      date: "2022-09-01",
    });

    console.log(response);
  } catch (error) {
    console.error(error);
  }
}

main();

File: airport_on_time_performance/v1/get/Python SDK/airport_on_time_performance.py
# Install the Python library from https://pypi.org/project/waconvo
from waconvo import Client, ResponseError

waconvo = Client(
    client_id='YOUR_WACONVO_API_KEY',
    client_secret='YOUR_WACONVO_API_SECRET'
)

try:
    '''
    Will there be a delay in the JFK airport on the 1st of December?
    '''
    response = waconvo.airport.predictions.on_time.get(
        airportCode='JFK', date='2021-12-01')
    print(response.data)
except ResponseError as error:
    raise error
